Active Member Requirements (#2)
BUUPE/Governing-Documents: Constitution.md

# Article I: Name and Objective

1. The name of this organization shall be the Massachusetts Eta Chapter of Boston University of the Upsilon Pi Epsilon Association, an honor society in computing and information disciplines.

> 2. The objective of this chapter of the Association shall be the promotion of high scholarship and original investigation in the various fields of computational disciplines. Furthermore, this chapter will focus on preparing members to apply for and work in technical fields in professional.

Author: ```suggestion
2. The objective of this chapter of the Association shall be the promotion of high scholarship and original investigation in the various fields of computational disciplines. Furthermore, this chapter will focus on preparing members to apply for and work in technical fields in industry.
```

1. All candidates for election to membership, qualified according to the previous section, shall be reviewed by the Recruitment Committee. Those reported favorably by the Committee shall be proposed for elections.
2. Each candidate must attend one information session and two recruitment events, unless otherwise exempt through extenuating circumstances, determined by the Recruitment Committee.
3. Each candidate must attend an interview session, consisting of both a behavioral and technical interview.
4. Each candidate must be discussed individually by name in regards to social, professional, and technical merits. All candidates shall be voted on after all have been discussed.
5. Election of a candidate to membership shall be by three-fourths affirmative vote by the members present and voting.
6. The maximum number of members shall not be limited.
7. The election of new members shall be held within six weeks after the beginning of the Fall and Spring semesters.
8. The Director of Recruitment shall appoint a recruitment team which will oversee the membership process. 

# Article V: Officers

> 1. The Executive Board of the chapter shall consist of the Faculty Advisor and the following officers: President, Executive Vice-President, Secretary, Director of Recruitment, Director of Marketing, Director of Operations, Director of Internal Development, Treasurer, Secretary.

Author: ```suggestion
1. The Executive Board of the chapter shall consist of the Faculty Advisor and the following officers: President, Vice-President, Treasurer, Secretary, Director of Recruitment, Director of Marketing, Director of Operations, Director of Internal Development.
```

11. The Faculty Advisor shall be a staff member of an information discipline. The duties of the Faculty Advisor shall be to consult on matters of chapter policy and to act as auditor of chapter finances. 
12. Election of officers shall be held during a chapter meeting in the middle of each Spring semester. Newly elected officers shall be brought up to date for the remainder of the Spring semester and will begin to serve on the last meeting of the Spring semester.
13. Vacancies in any office other than the President shall be filled by appointment by the President with the approval of the Executive Board.
14. The Recruitment Committee shall consist of the Director of Recruitment, their team, and the President and Executive Vice-President. The Recruitment Committee shall be responsible for staffing and organizing the recruitment process.

# Article VI: Ratification and Amendment

1. This Constitution shall become effective immediately upon ratification by a three-fourths affirmative vote of the invited charter initiates.
2. This constitution shall remain in effect until revised or amended. The revision of the constitution will abide by the same constraints of amendments.

> 3. Amendments in this Constitution may be proposed by any member of the chapter at a regular or special meeting. The proposed amendment shall be discussed and a vote for further discussion shall be taken; a simple majority of the active members present shall determine that consideration. On approval, the proposed amendment shall be formally framed by the Executive Board.

Author: ```suggestion
3. Amendments in this Constitution may be proposed by any member of the chapter at a regular or special meeting. The proposed amendment shall be discussed and a vote for further discussion shall be taken; a two-thirds affirmative vote of the active members present shall determine that consideration. On approval, the proposed amendment shall be formally framed by the Executive Board.
```
